src/component/userGuess.jsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from "react";
+
+//Requirements
+//1. When the user guesses a letter, add that letter to the App component's guessedLetters state.
+
+export const AddGuess = ({
```

Comment:
Let's name component file names and the component itself the same. So file name here should be AddGuess